pkgs/tools/security/sheesy-cli/default.nix

```
@@ -0,0 +1,48 @@
+{ lib, stdenv, rustPlatform, fetchCrate, installShellFiles
+, libgpgerror, gpgme, gettext, openssl, darwin
+}:
+
+let
+
+  features = [ "vault" "extract" "completions" "substitute" "process" ];
+
+in
+
+rustPlatform.buildRustPackage rec {
+  pname = "sheesy-cli";
+  version = "4.0.11";
+
+  src = fetchCrate {
+    inherit version pname;
+    sha256 = "1l21ji9zqy8x1g2gvqwdhya505max07ibx1hh88s36k0jbvdb7xc";
+  };
+
+  cargoSha256 = "159a5ph1gxwcgahyr8885lq3c1w76nxzfrfdpyqixqrr7jzx2rd3";
+  cargoDepsName = pname;
+
+  cargoBuildFlags = [ "--bin sy" "--features" "${lib.concatStringsSep "," features}" ];
+
+  nativeBuildInputs = [ libgpgerror gpgme gettext installShellFiles ];
+
+  buildInputs = [ ]
+  ++ (lib.optionals stdenv.isLinux [ openssl ])
+  ++ (lib.optionals stdenv.isDarwin (with darwin.apple_sdk.frameworks; [
+    Security
+  ]));
```

```suggestion
  buildInputs = lib.optionals stdenv.isLinux [ openssl ]
    ++ lib.optionals stdenv.isDarwin [ Security ];
```
Please inherit Security from the top level entry.